src/subscription/cart/SubscriptionCart.jsx

```diff
@@ -0,0 +1,134 @@
+import { ORDER_TYPES } from '../../payment/data/constants';
+
+import BulkOrderSummaryTable from '../../payment/cart/BulkOrderSummaryTable';
+import CartSkeleton from '../../payment/cart/CartSkeleton';
+import SubscriptionContent from './content/SubscriptionContent';
+import CurrencyDisclaimer from '../../payment/cart/CurrencyDisclaimer';
+import OrderSummary from '../../payment/cart/OrderSummary';
+import ProductLineItem from '../../payment/cart/ProductLineItem';
+import SummaryTable from '../../payment/cart/SummaryTable';
+import TotalTable from '../../payment/cart/TotalTable';
+import UpdateQuantityForm from '../../payment/cart/UpdateQuantityForm';
+import SubscriptionLegal from './legal/SubscriptionLegal';
+import SubscriptionOrderDetails from './order-details/SubscriptionOrderDetails';
+
+/**
+ * SubscriptionCart component
+ * renders the cart details
+ */
+class SubscriptionCart extends React.Component {
+  renderCart() {
+    const {
+      products,
+      orderType,
+      isCurrencyConverted,
+      orderTotal,
+      summaryPrice,
+      summarySubtotal,
+      summaryQuantity,
+      loaded,
+    } = this.props;
+    const isBulkOrder = orderType === ORDER_TYPES.BULK_ENROLLMENT;
+    return (
+      <div>
+        <span className="sr-only">
+          <FormattedMessage
+            id="payment.screen.reader.cart.details.loaded"
+            defaultMessage="Shopping cart details are loaded."
+            description="Screen reader text to be read when cart details load."
+          />
+        </span>
+
+        <SubscriptionContent>
+          {products.map(product => (
+            <ProductLineItem
+              key={product.title}
+              {...product}
+            />
+          ))}
+
+          {isBulkOrder ? <UpdateQuantityForm /> : null}
+        </SubscriptionContent>
+
+        {loaded
+          ? (
+            <OrderSummary> {isBulkOrder ? (
+              <BulkOrderSummaryTable
+                price={summaryPrice}
+                subtotal={summarySubtotal}
+                quantity={summaryQuantity}
+              />
+            ) : (<SummaryTable price={summaryPrice} isSubscription />)}
+              <TotalTable total={orderTotal} isSubscription />
+
+              {isCurrencyConverted ? <CurrencyDisclaimer /> : null}
+            </OrderSummary>
+          ) : (
+            <>
+              <div className="skeleton py-2 mb-3 w-50" />
+              <div className="skeleton py-2 mb-2" />
+              <div className="skeleton py-2 mb-5" />
+            </>
+          )}
+        <SubscriptionOrderDetails />
+        <SubscriptionLegal price={summaryPrice} />
+      </div>
+    );
+  }
+
+  render() {
+    const {
+      intl,
+      loading,
+    } = this.props;
+
+    return (
+      <section
+        aria-live="polite"
+        aria-relevant="all"
+        aria-label={intl.formatMessage(messages['payment.section.cart.label'])}
+      >
+        {loading ? <CartSkeleton /> : this.renderCart() }
```

Comment:
We'll need to modify this `CartSkeleton` component to reflect the number of items subs will have (not just one course) or create a `SubscriptionCartSkeleton`. Imo, just adding 2 extra skeletons for the "In Your Cart" section if `isSubscription` is true, will suffice

Reply:
hmm.. TBH I'm not a big fan of content have loading skeletons. first they make the page slow and unresponsive, secondly they should represent page layout semantics instead of replicating the full application content. 

However, we can't be sure the exact number of products/courses associated with each program, so my suggestion is to keep what we have here.

Reply:
I haven't seen what UX has, but if what you have matches it than disregard my comment!

You're right that we can't know the exact number of courses, good call.

Reply:
I just noticed Michelle updated designs for loading states what wasn't available earlier so will be updating skeletons in upcoming PR.